style.css

```diff
@@ -21,25 +25,129 @@ a {
   display: none;
 }
 
-header {
+.topnav {
+  overflow: hidden;
+  background-color: transparent;
   display: flex;
-  justify-content: space-between;
+  justify-content: center;
   align-items: center;
-  padding: 24px 0;
   height: 48px;
 }
 
-header .welcome {
-  font-size: 20px;
-  line-height: 28px;
-  margin-left: 24px;
-  color: #172b4d;
+.topnav a {
+  color: #344563;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
 }
 
-header img {
-  margin-right: 24px;
+@media screen and (min-width: 600px) {
+  .topnav {
+    justify-content: center;
+  }
+
+  .topnav a {
+    float: left;
+    display: block;
+    color: #344563;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .topnav a:hover {
+    background-color: #ddd;
+    color: black;
+  }
+
+  .topnav a.welcome {
+    display: none;
+  }
+
+  .topnav .icon {
+    display: none;
+  }
 }
 
+@media screen and (max-width: 600px) {
+  .rightNav {
+    font-size: 2rem;
+    font-weight: 600;
+    line-height: 2.75rem;
+    color: #3a4a42;
+  }
+
+  .topnav {
+    justify-content: space-between;
+  }
+
+  .topnav a:not(:first-child) {
+    display: none;
+  }
+
+  .topnav a.icon {
+    float: right;
+    display: block;
+  }
+
+  .topnav a.welcome {
+    display: block;
+    font-size: 20px;
+    line-height: 28px;
+    color: #172b4d;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .topnav.responsive {
+    position: relative;
+    display: block;
+    height: 755px;
+    border-bottom: 4px solid rgba(59, 59, 61, 0.08);
+    border-radius: 48px;
+    -webkit-border-radius: 48px;
+    -moz-border-radius: 48px;
+    -ms-border-radius: 48px;
+    -o-border-radius: 48px;
+    margin-bottom: 57px;
+  }
```

Comment:
- [ ] Your mobile menu is working fine, but it is possible for the user to scroll down when the menu is open making it disappear from the screen and a bad user experience. Kindly fix this issue, you could make the mobile menu fixed or disable the scrolling behavior when the menu is open.
 - Your design:
 - 
<img width="454" alt="image" src="https://user-images.githubusercontent.com/61919617/208719201-6fbf5cc0-3810-413f-85e7-aef7e4960d1c.png">

Reply:
Fixed.